oxd-server/src/main/java/org/gluu/oxd/server/Cli.java

```diff
@@ -293,4 +299,35 @@ private static Options options() {
 
         return options;
     }
+
+    private static boolean checkForMultipleOptions(CommandLine cmd){
+        int optionsCount = 0;
+        if (cmd.hasOption("l")) {
+            optionsCount ++;
+        }
+        if (cmd.hasOption("oxd_id")) {
+            optionsCount ++;
+        }
+        if (cmd.hasOption("d")) {
+            optionsCount ++;
+        }
+        if(optionsCount > 1)
+            return true;
+        else
+            return false;
```

Comment:
Formatting : Lets be consistent in code formatting. In line 308 we have `if` with space after it and curly brackets (`{}`) but in line 314 we have it without space and without brackets. I propose to always have space and always put brackets.

Reply:
Curly brackets added after if statement.